Gemfile

```diff
@@ -22,6 +22,7 @@ gem 'rsolr-ext', '~> 1.0.3'
 gem 'rubydora', '~> 2.0'
 gem 'uglifier', '>= 1.3.0'
 gem 'will_paginate', '~> 3.1', '>= 3.1.6'
+gem 'actionview', '>= 5.2.4.4'
```

Comment:
I don't think this needs to be added, if you do `bundle update rails` it should pick up the latest 5.2.4.4 without this addition.

Reply:
Good call. I fixed it. Thanks!